lib/test/command-acceptance.js

```diff
@@ -68,7 +68,7 @@ export default class CommandAcceptanceTest {
   run(options = {}) {
     return new Promise((resolve, reject) => {
       exec(`${ this.denaliPath } ${ this.command }`, {
-        env: Object.assign({}, process.env, options.env || {}),
+        env: Object.assign({}, options.env || {}, process.env),
```

Comment:
These shouldn't be switched. In this case, when running a command acceptance test, you _should_ be able to override the environment of that command from the test. The current process.env should just be the default "pass through" values otherwise.